<launch>
  
  <node pkg="camera" type="cameranode" name="quickcam_left" output="screen" respawn="true">
    <param name="width" type="int" value="1280" />
    <param name="height" type="int" value="960" />
    <param name="fps" type="int" value="30" />
    <param name="frame_id" type="string" value="leftcamera" />
    <param name="device_id" type="int" value="0" />
		<param name="camera_name" type="string" value="quickcam_left"/>
    <param name="cam_info_url" type="string" value="file://$(find camera)/camera_info/2015/quickcam_left_3.yaml"/>
    <remap from="/image_raw" to="mystereo/left/image_raw"/>
    <remap from="/camera_info" to="mystereo/left/camera_info"/>
  </node>

  <node pkg="camera" type="cameranode" name="quickcam_right" output="screen" respawn="true">
    <param name="width" type="int" value="1280" />
    <param name="height" type="int" value="960" />
    <param name="fps" type="int" value="30" />
    <param name="frame_id" type="string" value="leftcamera" />
    <param name="device_id" type="int" value="1" />
    <param name="camera_name" type="string" value="quickcam_right"/>
    <param name="cam_info_url" type="string" value="file://$(find camera)/camera_info/2015/quickcam_right_3.yaml"/>
    <remap from="/image_raw" to="mystereo/right/image_raw"/>
    <remap from="/camera_info" to="mystereo/right/camera_info"/>
  </node>

  <arg name="stereocamera" default = "mystereo"/>
  <node name="stereo_image_proc" pkg="stereo_image_proc" type="stereo_image_proc" ns="$(arg stereocamera)">
  	<param name="approximate_sync" type="bool" value="true"/>
  	<param name="queue_size" type="int" value="10"/>
  </node>

</launch>
